### PR TITLE
Add basic health check for Bamboo and Bitbucket

### PR DESCRIPTION
**Pull request by ungaralex**
### Description
I added some basic health checks for Bamboo and Bitbucket that request the status of the running servers and forward them to the client. You can now see them and any occurring error in the health panel for administrators.

image

### CODE REVIEW

File: src/main/webapp/i18n/de/health.json [hunk 1]
```diff
@@ -17,7 +17,9 @@
             "hystrix": "Hystrix",
             "diskSpace": "Festplattenspeicher",
             "mail": "Email",
-            "db": "Datenbank"
+            "db": "Datenbank",
+            "bamboo": "Bamboo",
```
Comment (krusche) [inline src/main/webapp/i18n/de/health.json]: What would happen if the server would respond a different CI and VCS system?
I think we could make this already a bit generic here

Comment (ungaralex) [reply src/main/webapp/i18n/de/health.json]: I now use generice CI and VCS names

**Comment by ungaralex** [comment]
> Looks good to me as well. Only thing I noticed is that you have different keys for German and English translations for `health.json` file. Can you fix it?

Ups, fixed that

File: src/main/java/de/tum/in/www1/artemis/service/connectors/ContinuousIntegrationServerHealthIndicator.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
+package de.tum.in.www1.artemis.service.connectors;
+
+import org.springframework.boot.actuate.health.Health;
+import org.springframework.boot.actuate.health.HealthIndicator;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+
+@Profile("bamboo")
```
Comment (krusche) [inline src/main/java/de/tum/in/www1/artemis/service/connectors/ContinuousIntegrationServerHealthIndicator.java]: does this really depend on bamboo? I think it should not!

File: src/main/java/de/tum/in/www1/artemis/service/connectors/VersionControlServerHealthIndicator.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
+package de.tum.in.www1.artemis.service.connectors;
+
+import org.springframework.boot.actuate.health.Health;
+import org.springframework.boot.actuate.health.HealthIndicator;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+
+@Profile("bitbucket")
```
Comment (krusche) [inline src/main/java/de/tum/in/www1/artemis/service/connectors/VersionControlServerHealthIndicator.java]: does this really depend on bitbucket? I think it should not!

File: src/main/webapp/i18n/de/health.json [hunk 4]
```diff
@@ -17,7 +17,9 @@
             "hystrix": "Hystrix",
             "diskSpace": "Festplattenspeicher",
             "mail": "Email",
-            "db": "Datenbank"
+            "db": "Datenbank",
+            "continuousIntegrationServer": "Continuous Integration Server",
```
Comment (krusche) [inline src/main/webapp/i18n/de/health.json]:
```suggestion
            "continuousIntegrationServer": "Kontinuierlicher Integrationsserver",
```

File: src/main/webapp/i18n/de/health.json [hunk 5]
```diff
@@ -17,7 +17,9 @@
...
+            "versionControlServer": "Version Control Server"
```
Comment (krusche) [inline src/main/webapp/i18n/de/health.json]:
```suggestion
            "versionControlServer": "Versionskontrollserver"
```

**Pull request merged**